index.html

```diff
@@ -48,6 +48,7 @@ <h1> Hiring </h1>
       <li> Muxable (Pion WebRTC + FFmpeg livestreaming) <a href="https://muxable.com"> muxable.com </a>, email <a href="mailto:kevin@muxable.com"> kevin@muxable.com </a> </li>
       <li> Daily (Mediasoup-based, JS/Typescript & Rust) <a href="https://www.daily.co/jobs"> https://www.daily.co/jobs </a> </li>
       <li> SignalWire (FreeSWITCH-based, C/C++, JS/Typescript) <a href="http://signalwire.com/jobs"> http://signalwire.com/jobs </a> </li>
+      <li> LiveKit (Open source, E2E WebRTC infrastructure) <a href="https://livekit.notion.site/Come-build-with-us-af2b57e9c4e64c1f84b745e6a5c6a9d1"> https://livekit.io/jobs </a> </li>
```

Comment:
@davidzhao The https://livekit.io/jobs link 404s - is there an alternate one you might want to use?

Reply:
heh, good catch. it's a notion doc at the moment. I fixed the href but not the text. doing that now.